wikipedia-dark.user.css

```diff
@@ -6546,6 +6547,49 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   .list-switcher-element a[role="button"] {
     color: var(--base-color) !important;
   }
+  tr > *[style*="background:#c0cfe4"],
+  tr > *[style*="background:#9999DF"] {
+    background-color: var(--blue-4) !important;
+  }
+  tr > *[style*="background:#a0ade3"],
+  tr > *[style*="background:#ccccff"] {
+    background-color: var(--blue-3) !important;
+  }
+  tr > *[style*="background:#e2e4c0"],
+  tr > *[style*="background:#d0cfa4"] {
+    background-color: var(--yellow-1) !important;
+  }
+  tr > *[style*="background:#c0d8e4"] {
+    background-color: var(--yellow-3) !important;
+  }
+  tr > *[style*="background:#e0e0b0"] {
+    background-color: var(--pink-1) !important;
+  }
+  tr > *[style*="background:#c0e4c0"],
+  tr > *[style*="background:#99cc99"],
+  tr > *[style*="background:#99CC99"] {
+    background-color: var(--green-3) !important;
+  }
+  tr > *[style*="background:#cfedcc"] {
+    background-color: var(--green-4) !important;
+  }
+  tr > *[style*="background:#e4d4c0"],
+  tr > *[style*="background:#f2caa4"] {
+    background-color: var(--red-3) !important;
+  }
+  tr > *[style*="background:#ECECEC"],
+  tr > *[style*="background:#C0C0C0"],
+  tr > *[style*="background:#d0d0d0"],
+  tr > *[style*="background:#EFEFEF"],
+  tr > *[style*="background: rgb(204, 204, 204)"],
+  tr > *[style*="background: #CCC"] {
+    background-color: var(--gray-4) !important;
+  }
+  div[style*="background:#d9ebff"],
+  tr > *[style*="background:#e0e0e0"],
+  tr > *[style*="background:#DEDEDE"]  {
+    background-color: var(--gray-3) !important;
+  }
```

Comment:
All in all its a good PR, I would instead of specifying `tr > *` probably try to make it a more general selector, this way we could try catch as many inline styles with these rules.

That is if at all possible.

other than previous comments I think upon chnages this would be good to go @AfroThundr3007730 for thoughts.

thx again @oowekyala for this good efforts.

Reply:
> All in all its a good PR, I would instead of specifying tr > * probably try to make it a more general selector, this way we could try catch as many inline styles with these rules.

That's true, maybe using just `*` would be ok?

Reply:
If it works